extensions/spring-web/deployment/src/main/java/io/quarkus/spring/web/deployment/ControllerAdviceAbstractExceptionMapperGenerator.java

```diff
@@ -104,89 +113,87 @@ protected void preGenerateMethodBody(ClassCreator cc) {
                     "Parameter type " + parameterTypes.get(notAllowedParameterIndex).name() + " is not supported for method"
                             + controllerAdviceMethod.name() + " of class" + controllerAdviceMethod.declaringClass().name());
         }
+
+        injectHttpHeadersField(cc);
+    }
+
+    private void injectHttpHeadersField(ClassCreator classCreator) {
```

Comment:
Minor complaint: I would call this `createHttpHeadersField`. The reason is that this part of the is actually creating a field in bytecode, it's not actually doing any injection which you obvisouly know happens at runtime.